README.md

```diff
@@ -1,75 +1,289 @@
 
-## Authors
 
-👤 **Author1**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+<!-- AUTHORS -->
 
-👤 **Author2**
+## 👥 Authors <a name="authors"></a>
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+> Mention all of the collaborators of this project.
 
-## 🤝 Contributing
+👤 **NWACHAN**
+
+- GitHub: [@Theophileaseh](https://github.com/Theophileaseh)
+
+👤 **CECILIA**
+
+- GitHub: [@c3c1l1a](https://github.com/c3c1l1a)
+
+👤 **MUHAMMAD**
+
+- GitHub: [@ShaheerCH](https://github.com/ShaheerCH)
+
+👤 **NADYA**
+
+- GitHub: [@petushka1](https://github.com/petushka1)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> We are lokking into improving the project and have planed several feature to implement in the future:
+
+- [ ] **[User profile editing]**
+- [ ] **[Doctor Role]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
 
-Give a ⭐️ if you like this project!
+## ⭐️ Show your support <a name="support"></a>
 
-## Acknowledgments
+> Write a message to encourage readers to support your project
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+If you like this project...
 
-## 📝 License
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **What is the best platform to Deploy the app built on top of API?**
+
+  - You can use any of the platforms that subbort your preferred tech. For instance, you can easily deploy your React app on Neflify
+
+- **How to get started with the project?**
+
+  - If you are building a fronted app and aim to use our basic API you can quickly start with fetching data from the endpointes provided. Read [API_DOC](http://localhost:3000/api-docs) to lean more. 
+  - If you aim to modify and extend our API you will need to clone repo and follow all setup (<a href="#getting-started">steps described above</a>)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template) grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪